base/system/format/format.c

```diff
@@ -435,8 +434,7 @@ int wmain(int argc, WCHAR *argv[])
             return -1;
 
         case DRIVE_NO_ROOT_DIR:
-            K32LoadStringW(GetModuleHandle(NULL), STRING_NO_VOLUME, szMsg, ARRAYSIZE(szMsg));
-            PrintWin32Error(szMsg, GetLastError());
+            ConResPuts(StdErr, STRING_ERROR_DRIVE_TYPE); // As STRING_NO_VOLUME has no ".\n".
```

Comment:
OK but actually it's still about the drive type, so one could argue that using `STRING_ERROR_DRIVE_TYPE` is actually more suitable than the previously-used error ID.
Maybe @archeYR could tell us why he used the NO_VOLUME here instead.
```suggestion
            ConResPuts(StdErr, STRING_ERROR_DRIVE_TYPE);
```

Reply:
> Maybe @archeYR could tell us why he used the NO_VOLUME here instead.

That was added in https://github.com/reactos/reactos/commit/d45e33685bc48a7620ed3fc01fb0a58b82218f04 , my suggestion was just about removing the error code printing.